astronomaly/utils/utils.py

```diff
@@ -77,7 +120,7 @@ def create_catalogue_spreadsheet(image_dataset, scores,
         0.016 degrees
     """
 
-    workbook = xlsxwriter.Workbook(filename)
+    workbook = xlsxwriter.Workbook(filename, {'nan_inf_to_errors': True})
```

Comment:
This removes NaNs from numeric data and writes a blank cell instead. On one occasion the function failed to write the file because of NaNs so I added this. Not sure if it is actually needed since NaNs should not be returned (the one I got was due to another error that I fixed).